Cargo.toml

```diff
@@ -15,5 +15,8 @@ members = [
     "tracing-serde",
     "tracing-appender",
     "tracing-journald",
+    "tracing-test",
```

Comment:
I worry about this name. Can your consider something else? We might want to use this for our internal test fixtures _or_ a library that certain statements are logged.

Reply:
> a library that certain statements are logged

The current `#[traced_test]` macro already does this...

I think it's the most fitting name, it both enables easy logging in tests and enables assertions. The available assertions could be expanded. From a user point of view, the user-facing crate name is more important than the name of internal test tools. For internal test fixtures, the crate could be called `tracing-testfixtures`.

If you disagree, could you suggest another name for the to-be-published tracing testing crate? :slightly_smiling_face: